velox/functions/sparksql/tests/MapTest.cpp

```diff
@@ -105,5 +106,122 @@ TEST_F(MapTest, ErrorCases) {
       {inputVector1, inputVector2, inputVector11, inputVector22},
       true);
 }
+
+TEST_F(MapTest, complexTypes) {
+  auto rowKey = makeRowVector({
+      makeFlatVector<int64_t>(1, [](auto row) { return row; }),
+  });
+  auto rowValue = makeRowVector({
+      makeFlatVector<int64_t>(1, [](auto row) { return row; }),
+  });
+  auto mapRowResult = makeMapVector(
```

Comment:
This code takes a lot of lines and repeats many times. Consider extracting helper function:

```
auto mapRowResult = makeSingleMap(rowKey, rowValue);
```

Reply:
Updated, thanks.